tests/index.js

```diff
@@ -127,5 +127,19 @@ describe('git-repo-info', function() {
         assert.deepEqual(result, expected);
       });
     }
+	
+	it('returns an object with repo info, including the full branch name, if the branch name includes any slashes', function() {
```

Comment:
I *think* I fixed the whitespace issue. At least, I made sure that the lines I added looked the same as what was already there in terms of whitespace and indenting.